### PR TITLE
fixing the ignored scenarios in conditions

### PR DESCRIPTION
**Pull request by mherman22**
**Ticket ID**
https://issues.openmrs.org/browse/RATEST-242.

**Description**
The purpose of this pull request is to address the previously ignored scenarios in the conditions.feature file.

**Relation**
It is also related to the following which were closed or merged to pave way for this one: 

1. https://github.com/openmrs/openmrs-contrib-qaframework/pull/163
2. https://github.com/openmrs/openmrs-distro-referenceapplication/pull/582
3. https://github.com/openmrs/openmrs-contrib-qaframework/pull/142
cc: @kdaud @sherrif10

### CODE REVIEW

File: src/test/java/org/openmrs/contrib/qaframework/automation/ConditionsSteps.java [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-import io.cucumber.java.Before;
-import io.cucumber.java.en.And;
-import io.cucumber.java.en.Given;
-import io.cucumber.java.en.Then;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
 import org.apache.commons.lang3.StringUtils;
 import org.openmrs.contrib.qaframework.RunTest;
 import org.openmrs.reference.page.ConditionPage;
```
Comment (sherrif10) [inline src/test/java/org/openmrs/contrib/qaframework/automation/ConditionsSteps.java:8]: Change the import statemement package to import from https://github.com/openmrs/openmrs-contrib-qaframework/tree/master/src/test/java/org/openmrs/contrib/qaframework/page to cater for the current changes

Comment (mherman22) [reply src/test/java/org/openmrs/contrib/qaframework/automation/ConditionsSteps.java:8]: alright thanks


> to cater for the current changes

Comment (mherman22) [reply src/test/java/org/openmrs/contrib/qaframework/automation/ConditionsSteps.java:8]: hey @sherrif10 , the changes by [this Pull request](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/582) are not captured here-->  https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/src/test/java/org/openmrs/contrib/qaframework/page/ConditionsPage.java

**Comment by mherman22** [comment]
> Can you first git log and share the screen shot
Screenshot from 2021-10-18 14-08-33

**Comment by kdaud** [comment]
@mherman22 which command are you using to update your branch with changes from master?

**Comment by mherman22** [comment]
> @mherman22 which command are you using to update your branch with changes from master?

`**git pull --rebase upstream master**`

**Comment by mherman22** [comment]
@kdaud youre a life saver...
thanks

> > **git pull --rebase upstream master**
> 
> The command above pulls master changes from upstream which brings out the concern [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/164#issuecomment-945646839).
> 
> @mherman22 did you ever get a chance to look at this link? https://gist.github.com/whoisryosuke/36b3b41e738394170b9a7c230665e6b9

**Pull request closed without merging**